contracts/sol6/Dao/KyberStaking.sol

```diff
@@ -201,11 +200,13 @@ contract KyberStaking is IKyberStaking, EpochUtils, ReentrancyGuard {
     }
 
     /**
+     * @notice return raw data of a staker for a specific epoch
+     *         WARN: data may not be accurate if data has not been inited
```

Comment:
WARNING: should be used only for init data. So use only for a staker that didn't delegate.

so its clear its not for pool operator.

and have same warning in interface.